types/passport/tslint.json

```diff
@@ -3,6 +3,7 @@
     "rules": {
         "no-unnecessary-generics": false,
         "no-void-expression": false,
+        "no-empty-interface": false,
```

Comment:
Without this, there is always a tslint error, saying to use either an explicit type or an interface.

Reply:
Do not disable it for the whole package, fix it at the location by adding tslint disable comment. Refer https://github.com/DefinitelyTyped/DefinitelyTyped#lint

Reply:
Done.